scripts/crdb_restore.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+BACKUP=$1
+if [[ $BACKUP == "" ]]; then
+  echo "No backup name given. It should look like '2020-01-29'."
+  exit 1
+fi
+
+# Set the environment:
```

Comment:
I am assuming you are setting the environment to ensure the access key id and stuff is set, perhaps it makes sense to explicitly check for those and exit with an informative user message.

Reply:
Sure, I can add that.